sections/quick-order-form.liquid

```diff
@@ -193,12 +224,29 @@
 
 {% schema %}
 {
-  "name": "Quick Order form",
+  "name": "t:sections.quick-order-form.name",
+  "limit": 1,
+  "enabled_on": {
+    "templates": ["index", "list-collections", "page", "blog", "article"]
```

Comment:
Let's confirm with @nicklepine to ensure these are the correct templates

Reply:
Confirmed!